Cython/Compiler/FlowControl.pxd

```diff
@@ -35,6 +35,7 @@ cdef class NameAssignment:
     cdef public object bit
     cdef public object inferred_type
     cdef public object rhs_scope
+    cdef public bint in_parallel
```

Comment:
We might want to move all `bint` attributes together, to the end. I don't think it would make a difference for alignment, but it also seems unnecessary to keep them in more than one place.